TypeCobol.LanguageServer.Test/ProcessorTests/Completion/AfterOf/NoResult.txt

```diff
@@ -0,0 +1,31 @@
+       IDENTIFICATION DIVISION.
+       PROGRAM-ID. TCOBCOMP.
+       DATA DIVISION.
+       WORKING-STORAGE SECTION.
+      * should suggest nothing!
```

Comment:
Yes it is not supported. Could you remove the

```
            if (tokenBeforeOf == null || !(tokenBeforeOf.TokenType == TokenType.ADDRESS || tokenBeforeOf.TokenType == TokenType.UserDefinedWord))
                return completionItems;
```

in `CompletionForOf` (+ add a `?` after `tokenBeforeOf` in the switch) ?